db/fixtures/tactical_vote_stt_recs.rb

```diff
@@ -31,31 +41,48 @@ def load
       rec = Recommendation.find_or_initialize_by(rec_key)
 
       source_advice = row[:advice]
-      advice_is_not_party = ACCEPTABLE_NON_PARTY_ADVICE.include?(source_advice)
 
-      if advice_is_not_party
-        rec.text = ADVICE_TRANSLATION[source_advice] || source_advice
-        # rubocop:disable Lint/UselessAssignment
-        party_short_code = nil
-        acceptable = !rec.text.nil?
-      else
-        rec.text = source_advice
-        party_short_code = rec.party_short_code_from_text
-        acceptable = !party_short_code.nil?
-      end
+      # ------------------------------------------------------------------------
 
-      if rec.text == "" || !acceptable
+      canonical_advice = source_advice.downcase.parameterize(separator: "_").to_sym
```

Comment:
Seems fine although I'm somewhat curious why you just didn't take `source_advice` unmodified as the keys for `SMV_CODES_BY_RECOMMENDATION_TEXT`.

Reply:
I could have done, but some of these imports are inconsistent in case, which would lead to more entries in the lookup. Also, a lookup hash with symbols rather than double-quoted text always looks cleaner to me.

Just preferences 🤷

Reply:
Thanks for the explanation!